examples/share_encrypted_data_with_partners/testing_only_dek.bin.index

```diff
@@ -0,0 +1 @@
+��5�iu��?HS�E����B�J�xi%Ϙ
```

Comment:
I added this hard-coded sample DEK in order to avoid adding a null resource with `openssl rand` command.
(`.index` at the end is to avoid errors on lint about whitespace at the end of the file - we can't add a whitespace in DEK)